src/c++/perf_analyzer/infer_context.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright 2023-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
> @rmccorm4: 2. The example report you posted looks great! However, it looks like it should also include some information to associate the responses with their requests. Are there plans to include either the corresponding inputs or request_ids etc. to correlate them later? Do we think that would be useful?

Yes, we're planning on including input information in the dump in the future. It was too technically complex for the first deadline.